### PR TITLE
Side nav changes

### PR DESCRIPTION
**Pull request by h-kataria**
- Move our apps in a collapsible “Automation Analytics” section
- changed our urls from ansible/insights/ to ansible/automation-analytics

https://issues.redhat.com/browse/AA-1114

@ryelo @Hyperkid123 please review

### CODE REVIEW

File: chrome/ansible-navigation.json [hunk 1]
```diff
@@ -180,34 +162,59 @@
             ]
         },
         {
-            "groupId": "business",
-            "icon": "trend-up",
-            "title": "Business Insights",
+            "groupId": "automation-analytics",
+            "icon": "",
```
Comment (Hyperkid123) [inline chrome/ansible-navigation.json]: An empty string is not allowed. Delete the attribute if you don't want any icons.

**Review by Hyperkid123 — COMMENTED** [review]
There was an issue with an invalid node version in the CI env. I have added a commit that specifies the node version.

**Review by Hyperkid123 — APPROVED** [review]

**Comment by h-kataria** [comment]
@Hyperkid123 with the link change in this PR, how do we handle the bookmarks with old links/bookmarks with `/insights/` in them, any pointers on that will be helpful.

**Comment by Hyperkid123** [comment]
@h-kataria  @astrozzc should be able to help you create redirects for akamai

**Comment by astrozzc** [comment]
Yea, just create a task in infra dashboard: https://issues.redhat.com/secure/RapidBoard.jspa?rapidView=6643&view=planning&selectedIssue=RHCLOUD-19056&issueLimit=100

**Comment by h-kataria** [comment]
@astrozzc created https://issues.redhat.com/browse/RHCLOUD-19061

**Pull request merged**